src/Parser.js

```diff
@@ -47,7 +47,7 @@ var ParseError = require("./ParseError");
  */
 function Parser(input, settings) {
     // Make a new lexer
-    this.lexer = new Lexer(input);
+    this.gullet = new MacroExpander(input, settings && settings.macros);
```

Comment:
I kind of like the `gullet`.  It's something that I think is unique to the way TeX operates and including that terminology in KaTeX is a great way to pay homage.  ^_^